### ISSUE TITLE
Send greetings from co-orgs to community

### ISSUE THREAD

**Issue by nickttng** [A11YChi/meetups-and-events#12]
**Why:** As a co-organizer, I want to keep in touch with the A11YChi community, sharing updates:

- Invite people to #a11ychi on web-a11y Slack (Send their email via a11ychi.meetup@gmail.com or DM in Meetup)
- Looking for virtual presenters

**Comment by nickttng** [comment]
The [message has been sent out](https://www.meetup.com/a11ychi/messages/boards/thread/52741469/0/#132498654), so this issue can be closed.
